### ISSUE TITLE
Empty address in log

### ISSUE THREAD

**Issue by yulongaion** [aionnetwork/aion#717]
I got a log with address empty, which fails the web3 client; however, the explorer is displaying a correct address, https://mastery.aion.network/#/transaction/a18d9b7d1b6f6c0209088fadfd4487a3cf623e3b3d4559abcce26b6b1934fda8. 

```
aion-127.0.0.1:8545> eth.getTransactionReceipt("0xa18d9b7d1b6f6c0209088fadfd4487a3cf623e3b3d4559abcce26b6b1934fda8")
{ blockHash: '0xadc75e6134978a472e4a42255aab0ea33a0cd85ed56633423414cfb79b13028c',
  nrgPrice: '0x02540be400',
  logsBloom: '00000000000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000000080000010000000000000000002000000000000000000000000000000000000000000',
  nrgUsed: 249417,
  contractAddress: '0xa06f2d4682cd64234d7aa644da23fa5edccc9e3c5bdf888f9a6db7f9128d344f',
  transactionIndex: 0,
  transactionHash: '0xa18d9b7d1b6f6c0209088fadfd4487a3cf623e3b3d4559abcce26b6b1934fda8',
  gasLimit: '0x0f4240',
  cumulativeNrgUsed: 249417,
  gasUsed: '0x03ce49',
  blockNumber: 1023871,
  root: '7e8c4e89a68b07f918a526d6368fdac421835f158155e50b41157b8cd4671fdb',
  cumulativeGasUsed: '0x3ce49',
  from: '0xa005bebbea7fc4475f2655cd3485f99db221c692e388052f345c628ebb028534',
  to: '0x',
  logs: 
   [ { address: '0x',
       logIndex: 0,
       data: '0x0000000000000000000000000000000100000000000000000000000000000030000000000000000000000000000000500000000000000000000000000000000361626300000000000000000000000000000000000000000000000000000000020000000000000000000000000000000100000000000000000000000000000002',
       topics: [Array],
       blockNumber: 1023871,
       transactionIndex: 0 } ],
  gasPrice: '0x02540be400',
  status: '0x1' }
```

**Comment by yulongaion** [comment]
Was digging into this issue a bit, and found the issue can be pinpointed to https://github.com/aionnetwork/aion/blob/843761824132b2dfe2a6e8dbc5b755167f571718/modApiServer/src/org/aion/api/server/types/TxRecptLg.java#L59

I think the above condition check is unnecessary, because an account creation transaction (`tx.getTo() == null`) can call another contract which emits a log.

**Comment by yulongaion** [comment]
Fixed by #656
